lib/netsuite/actions/add.rb

```diff
@@ -77,7 +77,7 @@ def add(credentials={})
           @errors = response.errors
 
           if response.success?
-            @internal_id = response.body[:@internal_id]
+            @internal_id = response.body[:@internal_id] if response.body.respond_to? :[]
```

Comment:
@ahuth Sorry I wasn't clear. `response.success` is true for media upload and every other type when saving, but it does not have `response.body` so that returns `nil` which makes `response.body[:@internal_id]` fail. I've only seen this be the case with media uploading, but this felt like a catch all solution for future cases as well.